apm/php: release 0.59.0 (#10407)
DataDog/documentation: content/en/tracing/setup_overview/compatibility_requirements/php.md

#### Deep call stacks on PHP 5

The call stack is limited on PHP 5. See the [deep call stack troubleshooting page][4] for more details.

### Generators

Instrumenting [generators][5] is not supported on PHP 5 and PHP 7.

### PCNTL

> We currently do not offer support for tracing processes forked via [pcntl][6]. When a call to `pcntl_fork` is detected, we disable tracing in the forked process. The main process will still be traced. We plan to provide support for `pcntl` in the future.

Reviewer: ```suggestion
We do not offer support for tracing processes forked using [pcntl][6]. When a call to `pcntl_fork` is detected, we disable tracing in the forked process. The main process is still be traced.
```

> If the application invokes `pcntl_unshare(CLONE_NEWUSER);` and the tracer is installed, the application will fatally crash. This happens because `unshare` with `CLONE_NEWUSER` requires the process [not to be threaded][7], while the PHP tracer uses a separate thread to send traces to the Datadog agent without blocking the main process.

Reviewer: ```suggestion
If the application invokes `pcntl_unshare(CLONE_NEWUSER);` and the tracer is installed, the application will fatally crash. This happens because `unshare` with `CLONE_NEWUSER` requires the process [not to be threaded][7], while the PHP tracer uses a separate thread to send traces to the Datadog Agent without blocking the main process.
```
